public/docs/getting_started/client.md

```diff
@@ -8,7 +8,7 @@ There are many Ethereum clients to choose from. We recommend different clients d
 
 We recommend [Ganache](/ganache), a personal blockchain for Ethereum development that runs on your desktop. Part of the Truffle Suite, Ganache simplifies dapp development by placing your contracts and transactions front and center. Using Ganache you can quickly see how your application affects the blockchain, and introspect details like your accounts, balances, contract creations and gas costs. You can also fine tune Ganache's advanced mining controls to better suit your needs. Ganache is available for Windows, Mac and Linux, and you can [download it here](/ganache).
 
-Ganache, when launched runs on `http://localhost:7545`. It will display the first 10 accounts and the mnemonic used to create those accounts. ([Read more about account mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).)
+Ganache, when launched runs on `http://127.0.0.1:7545`. It will display the first 10 accounts and the mnemonic used to create those accounts. ([Read more about account mnemonics](https://github.com/bitcoin/bips/blob/master/bip-0039.mediawiki).)
```

Comment:
Might also be good to add clarity on why `127.0.0.1` and not `localhost` here, too...

Same goes for anywhere else that we're potentially telling the user the URL for the first time.

Reply:
I think the the note (see below) is odd here. It's almost like we're introducing a topic and warning them about everything that can go wrong before they get a grasp of it. A newcomer wouldn't even know to swap `127.0.0.1` with `localhost`, and instead will input the URL in the address bar as they're told. 

If you think we definitely should keep it, can we pare it down? See example I gave in a comment above.

Reply:
Removed from here...